app/hub.py

```diff
@@ -371,9 +383,12 @@ def send_tx(self, amount, address, payment_id, priority, mixin, tx_desc, save_ad
                         
             msg = "<b>Coins successfully sent in the following transaction(s):</b><br><br>"
             for i in range(len(ret['tx_hash_list'])):
-                msg += "- Transaction ID: %s <br>  - Amount: %s <br>  - Fee: %s <br><br>" % (ret['tx_hash_list'][i], \
-                                                    print_money2(ret['amount_list'][i]), \
-                                                    print_money2(ret['fee_list'][i]))
+                if sweep_all:

```

Comment:
Why should we show less tx info for `sweep_all`?

Reply:
it think the rpc server only provide less information, see: https://getmonero.org/resources/developer-guides/wallet-rpc.html#sweep_all